### DISCUSSION TITLE
How do I determine what base model is used when calling the txt2img API route?

### DISCUSSION THREAD

**Discussion by monis24** [Mikubill/sd-webui-controlnet#576] [Q&A]
With the txt2img route, I understand that I can choose which controlnet model I'm using with the `controlnet_model` attribute, but where is it determined which base model (sdv1.5, my own trained model, etc.) is used?

**Comment by ljleb** [comment]
The `/controlnet/*2img` routes inherit the code from the original `/sdapi/v1/*2img` routes. you can use the same mechanisms that are available under `/sdapi/v1`.

Note that as the webui has been updated to support alswayson scripts. We are currently working on updating the code to deprecate `/controlnet/*2img` routes.
